src/resources/views/channel/show.blade.php

```diff
@@ -5,10 +5,41 @@
 @stop
 
 @push('page-actions')
-    <x-appshell::standard-actions :model="$channel" route="vanilo.admin.channel" :name="$channel->name" />
+    <x-appshell::standard-actions :model="$channel" route="vanilo.admin.channel" :name="$channel->name"/>
 @endpush
 
 @section('content')
+    <div class="row mb-4">
+        <div class="col">
+            <x-appshell::card-with-icon icon="channel" type="info">
+                {{ $channel->name }}
+                <x-slot:subtitle>
+                    {{ $channel->slug }} | {{ $channel->domain ? __('domain: :domain', ['domain' => $channel->domain]) : __('No domain assigned') }}
+                </x-slot:subtitle>
+            </x-appshell::card-with-icon>
+        </div>
+
+            <div class="col">
+                <x-appshell::card-with-icon icon="globe" type="success">
+                    {{ $country?->name ?? __('No country assigned') }}
+
+                    <x-slot:subtitle>
+                        {{ ($channel->currency ?? config('vanilo.foundation.currency.code')) ?: ($channel->language ?? substr(app()->getLocale(), 0, 2)) }}
+                    </x-slot:subtitle>
+                </x-appshell::card-with-icon>
+            </div>
+
+        <div class="col">
+            <x-appshell::card-with-icon icon="bag">
+                {{ __(':num purchases', ['num' => $orderCount]) }}
+
+                <x-slot:subtitle>
+                    {{ $ordersPerCurrency->map(fn($row) => "{$row->currency}: {$row->order_count}")->implode(' | ') }}
```

Comment:
I suggest using a @foreach loop and render as this: {{$row->total}} {{$row->currency}}

In general, please use as little "PHP magic" in blades as possible